app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated")
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():

```

Comment:
Pay attention that in production settings, when you are unsure about the incoming dict structure, it would be better to prefer `.get()` over `[]`.